lib/command.rb

```diff
@@ -60,6 +80,17 @@ def execute
          print_info(guest_ip, port, secrets_path, machine.index_uuid)
         end
       end
+
+      def print_boxversion(machine)
+        # Print the version of the underlying virtual machine
+	cmd = "test ! -f /etc/os-release || cat /etc/os-release"
+        version = ""
+        machine.communicate.execute(cmd) do |type, data|
+          version << data.chomp if type == :stdout
+        end
+        @env.ui.info(version)
+      end
+
```

Comment:
We should put a concrete logic here for getting ADB variant version. Just not any version as os-release file can have many versions